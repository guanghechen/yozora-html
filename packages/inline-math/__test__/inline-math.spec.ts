import type { InlineMath } from '@yozora/ast'
import renderInlineMath from '../src'

describe('snapshot', function () {
  it('basic', function () {
    const node: InlineMath = {
      type: 'inlineMath',
      value: '$x^2 + y^2 = z^2$',
    }
    expect(renderInlineMath(node)).toMatchSnapshot()
  })
})
