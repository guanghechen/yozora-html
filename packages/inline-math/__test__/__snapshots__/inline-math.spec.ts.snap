// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot basic 1`] = `"<span class=\\"yozora-inline-math\\">$x^2 + y^2 = z^2$</span>"`;
